File: js/stockPicker.js
exports.picker = function(prices) {
    let biggestSpread = 0
    let buySell = []
    for (day= 0; day, day < prices.length; day++) {   
        let sum = 0;
        for(let i =day+1 ; i < prices.length; i++){ 
            sum = (prices[i] - prices[day])
            if (sum > biggestSpread) {
                biggestSpread = sum;
                buySell[0] = day;
                buySell[1] = i;
            }
        }
    }
    return buySell
}

File: python/stock_picker.py
def picker(prices):
    # print(prices)
    i=0
    buy_sell = [0,0]
    biggest_spread = 0
    for day in range(0,len(prices)) :
        # print(day)
        i = day+1
        for i in range(day+1,len(prices)) :
           
            if i < len(prices):
                # print(prices[i],prices[day])
                sum = (prices[i]  - prices[day])
                
                if biggest_spread < sum:
                    biggest_spread = sum
                    buy_sell[0]= day
                    buy_sell[1]= i
                # print(sum)
                i+=1
    return(buy_sell)